packages/p2p/src/pages/orders/chat/__tests__/chat-wrapper.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ChatWrapper from '../chat-wrapper';
+import { isDesktop, isMobile } from '@deriv/shared';
+
+const mock_order_store = {
+    order_information: {
+        other_user_details: {
+            name: 'Test user',
+        },
+    },
+};
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(() => ({
```

Comment:
then just pass it here

```suggestion
    useStores: jest.fn(() => mock_store)
```